### ISSUE TITLE
PathStyleExtension dash feature is not working in low zoom level properly

### ISSUE THREAD

**Issue by mahdijirofti** [visgl/deck.gl#4857]
#### Description

My layer config is below and I have this issue with the dashed path layer.

![output](https://user-images.githubusercontent.com/22638025/89944745-920e2780-dc35-11ea-8e05-3669c00931e7.gif)

```
renderLayers() {
		const { data, getPath, getColor, getWidth } = this.props;
		return [
			new PathLayer(
				this.getSubLayerProps({
					id: 'path',
					data,
					pickable: false,
					widthScale: 2,
					widthMaxPixels: 8,
					widthMinPixels: 4,
					getPath,
					getColor,
					getWidth,
					rounded: true,
					dashJustified: true,
					extensions: [new PathStyleExtension({ dash: true })],
					getDashArray: [5, 5],
				}),
			),
		];
	}
```
#### Environment (please complete the following information):
 - Framework Version: deck.gl 8.2.2
 - Browser Version:  Chrome 79.0.3945.88 (Official Build) (64-bit)
 - OS: Ubuntu 18.04

#### Logs
I don't see any error in the console.

**Comment by mahdijirofti** [comment]
@Pessimistress 
Sir, do you have any idea?
Thank you very much.

**Comment by Pessimistress** [comment]
This is a known limitation of the extension. Dash is calculated per line segment, not the entire path. So when the segments are small you can't see the gaps.

We have a backlog work item to fix this, currently no ETA. @twojtasz 

A work around in the current release is to switch to a simpler path at low zoom levels. You can generate it with [@turf/simplify](https://turfjs.org/docs/#simplify).

**Comment by Pessimistress** [comment]
A new [highPrecisionDash](https://github.com/visgl/deck.gl/blob/master/docs/api-reference/extensions/path-style-extension.md#constructor) option is published in 8.3.0-alpha
